common/persistence/src/main/resources/db/migration/V9__alterMetadataTables.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE key_submission_metadata_with_client_metadata
+  ADD submitted_with_check_ins BOOLEAN DEFAULT NULL;
```

Comment:
For the most new columns it is required that they are set to null for the existing records. Please double check if:
- The constraint on these columns are also reflected in the code

`ALTER TABLE key_submission_metadata_with_client_metadata
  ADD submitted_with_check_ins BOOLEAN DEFAULT NULL;` here the column is set explicitly to null but for the other values not. Is there a particular reason for that? For example 
`ALTER TABLE test_result_metadata
  ADD pt_risk_level INTEGER,
  ADD pt_days_since_most_recent_date_at_risk_level INTEGER,
  ADD pt_hours_since_high_risk_warning INTEGER;` the risk level should also be null. Maybe lets have a talk about that :)